safe/definitions.py

```diff
@@ -55,6 +55,26 @@
     'functions. The area may have either <b>low</b>, <b>medium</b>, or '
     '<b>high</b> classification for the hazard.')
 
+concept_affected = tr(
+    '**Affected**: An exposure element (e.g. people, roads, buildings, land '
```

Comment:
Note that we can't use markdown here (so no **). You can use <b></b> html tags but I would suggest to avoid that too wherever possible